fud2/src/main.rs

```diff
@@ -241,7 +251,7 @@ fn build_driver() -> Driver {
         e.config_var_or("cycle-limit", "sim.cycle_limit", "500000000")?;
         e.rule(
             "verilator-compile",
-            "$verilator $in $testbench --trace --binary --top-module TOP -fno-inline -Mdir $out-dir",
+            "$verilator $in tb.sv --trace --binary --top-module TOP -fno-inline -Mdir $out-dir",
```

Comment:
Yep, this aspect is definitely worse about the new way. (But necessary, in a sense: if the files are going to be embedded in the binary, there is no way to avoid giving them a name. I suppose we could put them in `/tmp` or something instead of in the build directory, but that has its own downsides, including making it hard to do an ad hoc "override" of an extracted file.)

It would be great to have some tooling here to make these rules easier to write, i.e., to "thread through" dependencies on these extracted files. I am not sure yet how to accomplish that.